lib/new_relic/agent/parameter_filtering.rb

```diff
@@ -34,7 +34,7 @@ def apply_filters(env, params)
       end
 
       def filter_using_rails(params, filters)
-        return params if RAILS_FILTER_CLASS.nil?
+        return params if !defined?(RAILS_FILTER_CLASS) || (RAILS_FILTER_CLASS.nil?)
```

Comment:
You might already be working on this, but I wanted to drop in a reminder that the if condition might be a good candidate for a private method! Maybe named something like, `rails_filter_class?`

Reply:
Does [this](https://github.com/newrelic/newrelic-ruby-agent/pull/1566/commits/6edc1e66b4d29b94c2556755955f4606068e119c) work? :)